Assets/BossRoom/Scripts/Client/UI/UnityServicesUIHandler.cs

```diff
@@ -27,7 +27,7 @@ void ServiceErrorHandler(UnityServiceErrorMessage error)
             var errorMessage = error.Message;
             if (error.AffectedService == UnityServiceErrorMessage.Service.Lobby)
             {
-                if ((error.OriginalException as LobbyServiceException).Reason == LobbyExceptionReason.LobbyConflict)
+                if (error.OriginalException is LobbyServiceException {Reason: LobbyExceptionReason.LobbyConflict})
```

Comment:
wait, how come we can get errors coming from lobby, but with not lobby exception?

Reply:
The default value for OriginalException is null. In LobbyServiceFacade, line 128 there is a case where a UnityServiceErrorMessage is instantiated and published without an exception.

Reply:
ah crap true